src/Pelagos/Bundle/AppBundle/Controller/UI/DatasetReviewController.php

```diff
@@ -380,7 +380,7 @@ private function createNewDatasetSubmission(DatasetSubmission $datasetSubmission
         $datasetSubmission = new DatasetSubmission($datasetSubmission);
         $reviewedBy = $this->getUser()->getPerson();
         $reviewStartTimeStamp = new \DateTime('now', new \DateTimeZone('UTC'));
-        $datasetSubmissionReview = new DatasetSubmissionReview($datasetSubmission, $reviewedBy, $reviewStartTimeStamp);
+        $datasetSubmissionReview = new DatasetSubmissionReview($reviewedBy, $reviewStartTimeStamp);
```

Comment:
Need to set DatasetSubmissionReview object to DatasetSubmission, and it needs to updated after the DatasetSubmissionReview is created